.coveragerc

```
@@ -0,0 +1,7 @@
+[run]
+omit = 
+    Babylon/utils/command_helper.py
+    Babylon/utils/dry_run.py
+    Babylon/utils/help.py
+    Babylon/utils/logging.py
+    Babylon/utils/typing.py
```

```suggestion
    Babylon/utils/typing.py
source = Babylon/utils
```
I would specify coverage to limit to the utils folder ?